### DISCUSSION TITLE
Only show unanswered questions (persist search filters?)

### DISCUSSION THREAD

**Discussion by chadwhitacre** [community/community#16788] [Discussions]
As a community manager/project maintainer, I want to stay on top of the queue of incoming questions and regularly work it to zero, answering all questions. When I browse to a Q&A category I only want to see unanswered, unlocked questions.

**Comment by collaborationbest** [comment]
I think you can use search query

 is:issue is:open is:unlocked no:assignee

**Comment by brunooboy** [comment]
You can get closer to what you want using a more refined search.

Try this:

is:issue is:open is:unlocked comments:0

**Comment by shortcut119** [comment]
That’s a great suggestion. I’ve run into the same situation while trying to browse unanswered questions in the Discussions tab.

It would definitely help if GitHub could persist the “unanswered” filter across page refreshes or when navigating between categories. Right now, the filter resets every time you leave or reload the page, which makes it harder to continuously help people who still need answers.

A few possible UX ideas that could make this smoother:

Persistent query parameter: Keep is:unanswered in the URL when filtering, so it remains applied even if you navigate away and come back.

User preference setting: Allow users who frequently answer questions to set “show unanswered by default” as a personal preference in their Discussions view.

Saved filters: Similar to saved searches on GitHub code search, users could save custom filter combinations (like category:Q&A is:unanswered sort:recent).

Implementing one of these would make community contribution much more efficient — it’s a small tweak that could encourage more people to help out consistently.

Curious to hear if others would find persistent filters useful too!

**Comment by omidmousavi** [comment]
To filter and view only unanswered questions in a GitHub Discussions Q&A category, you can use GitHub's built-in search filters directly in the search bar on the Discussions page. Here's how to do it correctly and efficiently:

1. **Navigate to the Discussions page** of your repository or organization (e.g., https://github.com/orgs/community/discussions or your repo's /discussions).

2. **Apply the filter for unanswered questions**:
   - In the search bar, enter `is:unanswered` (or combine with other filters like `is:open is:unlocked` for more precision).
   - For example: `is:unanswered is:open category:"Q&A"` (adjust the category name to match your setup, like "Q&A" or "Questions").
   - This will show only discussions that are unanswered (no accepted answer) and open.

3. **Persisting the filter**:
   - GitHub doesn't currently have a built-in user preference to default to "unanswered" or persist filters across sessions, but you can bookmark the URL after applying the filter (e.g., https://github.com/orgs/community/discussions?q=is%3Aunanswered+is%3Aopen). This way, you can quickly return to it.
   - As a workaround for persistence, use browser extensions like "Saved Searches" or GitHub's advanced search features to save custom queries.

4. **Additional tips**:
   - If you want to focus on questions with no comments at all, use `comments:0` in the search.
   - For assignee-related filters (e.g., no assignee), add `no:assignee`.
   - If this is a frequent need, consider suggesting a feature improvement via GitHub's feedback channels, as persistent filters or default views would enhance the experience for maintainers like you.
